cli/lib/kontena/cli/common.rb

```diff
@@ -22,6 +22,16 @@ def pastel
         @pastel ||= Pastel.new(enabled: $stdout.tty?)
       end
 
+      def stdin_input(message = nil)
+        if $stdin.tty?
+          prompt.multiline(message).join.chomp
```

Comment:
The `prompt.multiline` is a bit odd for things like `kontena vault write test`.... just writing something and hitting enter does not accept the value, you have to hit ^D.

Should this take some options to use different prompts depending on what is being prompted? Looking around in the [docs](https://github.com/piotrmurach/tty-prompt#23-multiline)...

* `prompt.multiline` for `kontena master config import` and `kontena vault import`?
* `prompt.mask` for `kontena vault write/update`?
* `prompt.ask` otherwise